ESDT-specs.md

```diff
@@ -0,0 +1,523 @@
+
+**[History](#history)**
+
+## Summary
+A standard interface for fungible, non-fungible (NFT) and semi-fungible (SFT) tokens that allows issuance, management and transfer of tokens, without the need for Smart Contracts (such as ERC20). ESDT tokens have native in-protocol support, so transactions with ESDT tokens do not require a VM at all. In effect, this means that ESDT tokens are as fast and as scalable as the native eGold token itself.
+
+## Motivation
+Elrond is a novel state sharded architecture. Moreover, Elrond offers a full Smart Contract development experience in such a way that Smart Contract developers do not need to know about sharding, address space, transaction broadcasting etc. 
+
+Running Smart Contracts is done through the Elrond VM, the fastest VM in the blockchain space that is running on WASM, is extra secure, has a built-in gas model, etc. On the other hand, we acknowledge that digital tokens such as stable coins will need to have the same scalability as the base layer protocol.
+
+With ESDT, we are proposing a novel token standard, where different digital tokens can work on top of the Elrond Network as fast as the native eGold token. This standard provides basic functionality to issue, manage, mint, burn, froze, transfer, etc. ESDT tokens.
+
+Users also do not need to worry about sharding when transacting ESDT tokens, because the protocol employs the same handling mechanisms for ESDT transactions across shards as the mechanisms used for the eGLD native token. Sharding is therefore automatically handled and invisible to the user.
+Technically, the balances of ESDT tokens held by an Account are stored directly under the data trie of that Account. It also implies that an Account can hold balances of any number of ESDT tokens, in addition to the native eGold balance. The protocol guarantees that no Account can modify the storage of ESDT tokens, neither its own nor of other Accounts.
+ESDT tokens can be issued, owned and held by any Account on the Elrond network, which means that both users and Smart Contracts have the same functionality available to them. Due to the design of ESDT tokens, Smart Contracts can manage tokens with ease, and they can even react to an ESDT transfer.
+
+## Fungible Tokens
+### Specification
+ESDT allows any tokens on Elrond network to be integrated by other applications and dApps: from wallets to decentralized exchanges.
+
+### Methods
+### Issuance of fungible ESDT tokens
+ESDT tokens are issued via a request to the Metachain, which is a transaction submitted by the Account which will manage the tokens. When issuing a token, one must provide a token name, a ticker, the initial supply, the number of decimals for display purpose and optionally additional properties. This transaction has the form:
+
+```
+IssuanceTransaction {
+    Sender: <account address of the token manager>
+    Receiver: erd1qqqqqqqqqqqqqqqpqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqzllls8a5w6u
+    Value: 5000000000000000000 (5 eGLD)
```

Comment:
Will remain 5 EGLD?

Reply:
For now we will start with 0.05 EGLD -> 50000000000000000 (0.05 eGLD)